json-endpoints/class.wpcom-json-api-publicize-scheduled-actions-endpoint.php

```diff
@@ -0,0 +1,112 @@
+<?php
```

Comment:
Both scheduled_actions and published_actions endpoint should not be in Jetpack, since this data is on wpcom side only.
It WILL work for jetpack, however:
- publicize action runs on WPCOM infrastructure
- auth tokens for appropriate services live in wpcom tables
etc.

Plus, this code is highly experimental and under development :)

This is also coded during a meetup, so please dont judge us :)